tests/Unit/PrefixerTest.php

```diff
@@ -724,5 +724,50 @@ public function testStaticNamespacedClassIsPrefixed()
 
         $this->assertEquals($expected, $result);
     }
+    
+    
+    
+    
+    /**
+     * Prefix namespaced classnames after return statement.
+     *
+     * @see https://github.com/BrianHenryIE/strauss/issues/11
+     */
+    public function testReturnedNamespacedClassIsPrefixed()
+    {
+
+        $contents = 'return \Carbon_Fields\Carbon_Fields::resolve';
+        $expected = 'return \BrianHenryIE\Strauss\Carbon_Fields\Carbon_Fields::resolve';
+
+        $config = $this->createMock(StraussConfig::class);
+
+        $replacer = new Prefixer($config, __DIR__);
+        $result = $replacer->replaceNamespace($contents, 'Carbon_Fields\Carbon_Fields', 'BrianHenryIE\Strauss\Carbon_Fields\Carbon_Fields');
+
+        $this->assertEquals($expected, $result);
+    }
+    
+    
+    /**
+     * Prefix namespaced classnames between two tabs and colon.
+     *
+     * @see https://github.com/BrianHenryIE/strauss/issues/11
+     */
+    public function testReturnedNamespacedClassIsPrefixed()
+    {
+
+        $contents = "		\Carbon_Fields\Carbon_Fields::service( 'legacy_storage' )->enable()";
+        $expected = "		\BrianHenryIE\Strauss\Carbon_Fields\Carbon_Fields::service( 'legacy_storage' )->enable()";
```

Comment:
```suggestion
        $contents = "		\\Carbon_Fields\\Carbon_Fields::service( 'legacy_storage' )->enable()";
        $expected = "		\\BrianHenryIE\\Strauss\\Carbon_Fields\\Carbon_Fields::service( 'legacy_storage' )->enable()";
```

Two backslashes in double-quotes.